fastly/fixtures/dictionary_items/get.yaml

```diff
@@ -9,22 +9,25 @@ interactions:
       Fastly-Key:
       - xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
       User-Agent:
-      - FastlyGo/0.4.3.dev (+github.com/fastly/go-fastly; go1.12)
-    url: https://api.fastly.com/service/7i6HN3TK9wS159v2gPAZ8A/dictionary/2PSSfhzn58nbURuobYGT3I/item/test-dictionary-item
+      - FastlyGo/1.0.0 (+github.com/fastly/go-fastly; go1.12.1)
+    url: https://api.fastly.com/service/17Cozi1mqUCWuTgyYvipeW/dictionary/2qpiTjYtwgTuLyB2N6vFAq/item/test-dictionary-item
     method: GET
   response:
-    body: '{"updated_at":"2019-05-08T22:01:02Z","service_id":"7i6HN3TK9wS159v2gPAZ8A","deleted_at":null,"item_value":"value","dictionary_id":"2PSSfhzn58nbURuobYGT3I","item_key":"test-dictionary-item","created_at":"2019-05-08T22:01:02Z"}'
+    body: '{"created_at":"2019-07-16T11:30:23Z","item_value":"value","dictionary_id":"2qpiTjYtwgTuLyB2N6vFAq","service_id":"17Cozi1mqUCWuTgyYvipeW","updated_at":"2019-07-16T11:30:23Z","item_key":"test-dictionary-item","deleted_at":null}'
     headers:
       Accept-Ranges:
       - bytes
+      - bytes
+      - bytes
       Age:
       - "0"
+      - "0"
```

Comment:
Same here!

Reply:
No worries I will go through a correct these.

Reply:
Removed the duplicate entries in the newly created fixtures.